main/guides/integration/chain-integration.md

```diff
@@ -9,13 +9,8 @@ This section points at relevant reference documentation for the underlying `cosm
 
 The Agoric Network currently uses `cosmos-sdk` v0.45. The general Cosmos documentation for this version can be [found here](https://docs.cosmos.network/v0.45/), including structure and`golang` documentation, and REST API documentation. 
 
-Use the [v0.45.1 version of the  REST API](https://v1.cosmos.network/rpc/v0.45.1) for accessing the chain. To use the "Try it out" functionality, change the Base URL to `agoric-api.polkachu.com`:
 
-
-| ![Alt name of image](./assets/cosmos-api.png) |
```

Comment:
that image was pretty handy - are you proposing to omit it?

Reply:
I honestly don't know whether keep or remove it - this page does not exist anymore though.

Reply:
ugh. I suppose that's worth its own issue:

 - https://github.com/Agoric/agoric-sdk/issues/9446

Reply:
@dckc reckon we can merge changes to the rest of the files?

Reply:
https://docs.cosmos.network/api ?